src/test/java/com/google/api/codegen/protoannotations/testdata/csharp_library.baseline

```diff
@@ -1224,11 +1224,11 @@ namespace Google.Example.Library.V1.Snippets
             // Create client
             LibraryServiceClient libraryServiceClient = await LibraryServiceClient.CreateAsync();
             // Make the request
-            PagedAsyncEnumerable<ListStringsResponse, string> response =
+            PagedAsyncEnumerable<ListStringsResponse, IResourceName> response =
```

Comment:
This is the same as what the GAPIC config v1 gives: https://github.com/googleapis/gapic-generator/blob/5bad4a26f0b4ffe26c7e0f4b5559482ad6c8af3f/src/test/java/com/google/api/codegen/gapic/testdata/csharp/csharp_library.baseline#L1237